### ISSUE TITLE
`git clone` + `npm install` produces dirty checkout

### ISSUE THREAD

**Issue by ryan-roemer** [FormidableLabs/radium#548]
With commit https://github.com/FormidableLabs/radium/commit/90f353b8c1845bed43b32bd2d5c9b6f8883d49b2 we now have a `prepublish` command that mutates `package.json`.

Unfortunately, there's a [`wontfix` bug with NPM](https://github.com/npm/npm/issues/3059) wherein `npm install` **also** runs `preublish`. This manifests with a situation where a fresh git clone + install (or any install) yields a **dirty** checkout:

```sh
$ git clone git@github.com:FormidableLabs/radium.git

$ cd radium

$ npm install
# ... SNIPPED ...
> radium@0.16.2 deps-fix /Users/rroemer/Desktop/TEMP_RADUM/radium
> babel-node ./npm-scripts/move-babel-to-dependencies.js
# ... SNIPPED ...

$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

    modified:   package.json

no changes added to commit (use "git add" and/or "git commit -a")
```

There is some [new discussion](https://github.com/npm/npm/issues/10074) of changed behavior / a new hook to address this situation in `npm v4` when that's released.

**Comment by zachhale** [comment]
Yuck, that looks super annoying.

**Comment by ianobermiller** [comment]
Oh right, forgot about that "feature" they are going to finally fix it in npm@4, but that doesn't really help us now. Ideas?

For now, I think making modification of Radium slightly less annoying is worth not having babel in published dependencies.

**Comment by ryan-roemer** [comment]
@ianobermiller -- This is a bit of a potentially inconvenient switch but if you went instead of a `prepublish` workflow doing a:

```
preversion
version
postversion
```

workflow, `preversion` doesn't have the bug that `prepublish` does.

_but_, you'd have to publish with `npm version`

**Comment by ianobermiller** [comment]
Fixed in 92046c8234ec19b77e86dfbf78ecec5740500f94.
